securedrop/journalist_templates/submission_preferences_saved_flash.html

```diff
@@ -0,0 +1,10 @@
+{# these are flashed messages for the logo upload file verifiaction #}
+{% with messages = get_flashed_messages(with_categories=True, category_filter=["submission-preferences-success"]) %}
+  {% for category, message in messages %}
+    {% set category_status = category[23:] %}
```

Comment:
I realize that it's a magic number also in `securedrop/journalist_templates/logo_upload_flashed.html` but can you add a comment explaining where the 23 comes from?

Reply:
a8b2dc6e42c8056487ca16d95001295e794c5529 I added the comment here. I think there's probably a better way to word this though, so if you have a suggestion let me know.